### PR TITLE
Calculator UI

### PR DESCRIPTION
**Pull request by omarsalem7**
I did the following in this PR:
- Delete all the boilerplate from CRA (text, images, styles).
- Create components directory, create a new Calculator.js file
- Create The UI for Calculator

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# React js Template
+# Math Magicians
```
Comment (TedLivist) [inline README.md:3]: - [x] Kindly remove unused boilerplate files as they make your codebase look neater

  public folder | src folder
  :---:|:---:
  omar-public | omar-src

File: src/App.js [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+import Calculator from './components/Calculator';
 
 function App() {
-  return <div className="App"><h1>Hello world</h1></div>;
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
 }
```
Comment (TedLivist) [inline src/App.js]: - [x] Your components should be class-based. This App component is function-based, kindly refactor a class-based component.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @omarsalem7,

Good job so far!
✔️ Good documentation
✔️ App UI fits template design
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
#### _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (omarsalem7) [reply src/App.js]: Hello @TedLivist 
I changes your suggestion, you can check it now 
have a nice day 😄

**Review by julie-ify — APPROVED** [review]
## Hi @omarsalem7,

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**